### PR TITLE
focus on blushy background tabs (fixes #93)

### LINKED ISSUE #93: opening a blushy site in a new tab doesn't quite work...
We might need to be a bit more careful handling things like ctrl+click and open in new tab (and probably even open in new window, but who does that?).

### PR DESCRIPTION
**Pull request by mozkeeler**

### CODE REVIEW

File: test/test-main.js [hunk 1]
```diff
@@ -311,6 +311,34 @@ function testWhitelistCategory() {
...
+  let key = bpUtil.getKeyForHost("localhost");
+  ss.storage.blushlist.map[key] = "testing";
+
+  // we have to clear these together to keep things consistent
+  delete ss.storage.whitelistedDomains[key];
+  delete ss.storage.whitelistedCategories["testing"];
+
+  gEvents = gEvents.concat([kEvents.BLUSHY_SITE]);
+  let promise = maybeShowConsentPanel(gUrl, false);
+  gAssert.ok(tabs.activeTab.index == 0, "active tab should be index 0");
```
Comment (monicachew) [inline test/test-main.js]: slightly more idiomatic to do
gAssert.equal(0, tabs.active.index, "active tab should be index 0")

File: test/test-main.js [hunk 2]
```diff
@@ -311,6 +311,34 @@ function testWhitelistCategory() {
...
+  tabs.open({ url: "about:blank",
+              inBackground: true,
+              onReady: function() { tabs[1].url = "http://localhost:4444/"; }
+            });
+  return promise.
+    then(function() { gAssert.ok(tabs.activeTab.index == 1,
```
Comment (monicachew) [inline test/test-main.js]: same here

**Comment by monicachew** [comment]
Does this work with ctrl-click, right-click, ctrl-enter from location bar, and is there any other tab opening method? :)

**Comment by mozkeeler** [comment]
It doesn't work from "open web location" (e.g. go to about:addons, use ctrl+l), but then again it never did. ctrl+enter doesn't seem to open in a new tab for me, even in normal Firefox.

**Comment by monicachew** [comment]
Ah, sorry, it's alt-enter 

----- Original Message -----

**Comment by mozkeeler** [comment]
looks like shift+alt+enter works

**Pull request merged**